src/app/containers/Video/helpers/fixtures.js

```diff
@@ -0,0 +1,556 @@
+              model: {
+                blocks: [
+                  {
+                    type: 'text',
+                    model: {
+                      blocks: [
+                        {
+                          type: 'paragraph',
+                          model: {
+                            text: 'Description of picture (ALT TEXT)',
+                            blocks: [
+                              {
+                                type: 'fragment',
+                                model: {
+                                  text: 'Description of picture (ALT TEXT)',
+                                  attributes: [],
+                                },
+                              },
+                            ],
+                          },
+                        },
+                      ],
+                    },
+                  },
+                ],
+              },
+            },
+          ],
+        },
+      },
+    ],
+  },
+  type: 'aresMedia',
+};
+
+export const videoNonUkBlock = {
+  type: 'aresMedia',
+  model: {
+    blocks: [
+      {
+        blockId: 'urn:bbc:ares::clip:p01kdbpz',
+        type: 'aresMediaMetadata',
+        model: {
+          id: 'p01kdbpz',
+          subType: 'clip',
+          format: 'audio_video',
+          title: "'I lost my leg to type 2 diabetes' (NON UK)",
+          synopses: {
+            short: "Diabetes type 2: 'I blame myself for the loss of my leg'",
+            long:
+              'Every week 170 people in the UK lose a limb because of diabetes. \nAround four million people in the UK are living with diabetes - and around 90% of them have type 2 diabetes, which is often linked to being overweight or inactive. \nDiabetes prescriptions are costing the NHS in England more than £1 billion a year, according to figures from NHS Digital.\nAnd a further 12 million people could be at risk of getting type 2 diabetes in the future.\nStephen Richardson has told the BBC he blames himself for the loss of his limb.',
+            medium:
+              'Stephen Richardson admits he ignored advice about diet and exercise.',
+          },
+          imageUrl: 'ichef.test.bbci.co.uk/images/ic/$recipe/p01kdbs1.jpg',
+          imageCopyright: 'BBC',
+          embedding: true,
+          advertising: true,
+          caption:
+            'Stephen Richardson had to have his leg amputated because of type 2 diabetes',
+          versions: [
+            {
+              versionId: 'p01kdbq2',
+              types: ['Original'],
+              duration: 122,
+              warnings: {},
+              availableTerritories: {
+                uk: false,
+                nonUk: true,
+              },
+              availableUntil: null,
+            },
+          ],
+          image: null,
+          syndication: {
+            destinations: [],
+          },
+        },
+      },
+      {
+        type: 'image',
+        model: {
+          blocks: [
```

Comment:
Again, altText and height and width values would be expected

Reply:
This is the exact data that Ares is giving us. 
Note that this image block is nested within the aresMedia block and is *not* an Optimo image block. This block is to be purely used to be passed to the Media Player.